Sources/SnowManLogic/Game.swift

```diff
@@ -0,0 +1,36 @@
+import Dictionary
+
+public class Game {
+	let dictionary: DictionaryManager
+	let view: View
+
+	var correctGuesses: [String] = []
+	var wrongGuesses: [String] = []
+	var gameWord: String = ""
+
+	public init(dictionary: DictionaryManager, view: View) {
+		self.dictionary = dictionary
+		self.view = view
+	}
+
+	public func getWord() -> String {
+		gameWord = dictionary.getRandomWord()
+		return gameWord
+	}
+
+	public func checkIfGuessingFullWord(guess: String) -> Bool {
+		if (guess.characters.count > 1) {
+			return true
+		} else {
+			return false
+		}
+	}
+
+	public func checkForGuessInWord(guess: Character, gameWord: String) -> Bool {
```

Comment:
This method doesn't use anything stored in the class, it might belong on something else, or maybe it could be a static method. (but it should use the stored game word variable rather than passing one in)